src/test/resources/application.properties

```diff
@@ -39,10 +31,10 @@ docs.trustedS3bucketName=trusted-bucket
 docs.trustedS3bucket=${docs.trustedS3bucketName}
 docs.trustedS3bucketKMSKeyId=123456
 
-clamav.root=http4://localhost:8086
+clamav.root=http4://localhost:9002
```

Comment:
This does run on 8086 in the main docker-compose for the project?

Reply:
This is to use the wiremock port in the integration tests